feature-libs/asm/occ/adapters/occ-asm.adapter.ts

```diff
@@ -29,7 +33,32 @@ export class OccAsmAdapter implements AsmAdapter {
   ) {
     this.baseSiteService
       .getActive()
-      .subscribe((value) => (this.activeBaseSite = value));
+      .subscribe((value) => (this.activeBaseSite = value)); // TODO: Concurrency issue?
+  }
+
+  customerLists(): Observable<CustomerListsPage> {
+    const headers = InterceptorUtil.createHeader(
```

Comment:
this header seems used in all functions in this adapter. Maybe make it as `protected` class variable.